Specify target path caching for resources.ExecuteAsTemplate (#2027)
gohugoio/hugoDocs: content/en/hugo-pipes/resource-from-template.md

In order to use Hugo Pipes function on an asset file containing Go Template magic the function `resources.ExecuteAsTemplate` must be used.

> The function takes three arguments: the target path for the created resource, the template context, and the resource object. The target path is used to cache the result.

Reviewer: ```suggestion
The result is cached using the target path as the cache key.
```

Author: I found it useful for it to explain what the first param is. It would be too bad to lose that.

Reviewer: Sorry, I meant to replace just the last sentence.

> The function takes three arguments: the target path for the created resource, the template context, and the resource object. The target path is used to cache the result.

Author: ```suggestion
The function takes three arguments: the target path for the created resource, the template context, and the resource object. The result is cached using the target path as the cache key.
```

Author: @jmooring How's that?

Reviewer: Thank you.
